core/templates/dev/head/domain/exploration/SolutionObjectFactory.js

```diff
@@ -67,6 +67,9 @@ oppia.factory('SolutionObjectFactory', [
         correctAnswer = '[Music Notes]';
       } else if (interactionId === 'LogicProof') {
         correctAnswer = this.correctAnswer.correct;
+      } else if (interactionId === 'FractionInput') {
+        correctAnswer = FractionObjectFactory
+            .fromDict(this.correctAnswer).toString();
```

Comment:
Generally we try to break at the '('. Could you rewrite as:

   correctAnswer = FractionObjectFactory.fromDict(
     this.correctAnswer).toString();

Thanks!

Reply:
Will do this. I think I used a similar line break elsewhere too. Not sure where though.

Reply:
Done. Also added the imports in collection_player and editor as FractionObjectFactory.js wasn't imported earlier

Reply:
Just to check, was that causing an error? If it wasn't, those imports are not needed.

Reply:
Yes. The collection page was broken without the import. It's fixed now 👍